### ISSUE TITLE
Training reproduce issue

### ISSUE THREAD

**Issue by amsword** [hujie-frank/SENet#48]
I tried to reproduce the training from scratch, but the accuracy is 5 point lower on Inception network. Can you share the solver file, e.g. number of iterations, learning rate policy, or any more details of the training?

**Comment by changlinzhang** [comment]
Hi, @amsword 
I also met the problem. Have you got any idea?

**Comment by hujie-frank** [comment]
You can employ the data augmentation strategy described [here](https://github.com/hujie-frank/SENet#augmentation) and decrease the learning rate by a factor of 10 manually when the validation loss plateau, rather taking step strategy with fixed epoches.

**Comment by DanielWang2099** [comment]
@hujie-frank 

Random Crop 8% ~ 100% 
may only crop 8% of the whole image during training. Is it a typo (80% ~ 100%)?

**Comment by hujie-frank** [comment]
It is not a typo, you can refer to "Training Methodology" in [_Going deeper with convolutions_](https://arxiv.org/pdf/1409.4842.pdf).
